### DISCUSSION TITLE
Tubeholder

### DISCUSSION THREAD

**Discussion by Domi1l** [cardonabits/haxo-hw#82] [Show and tell]
Hi, 
i designed a tubeholder, i dont liked it when i blow cold air against my hand. So i have to find a solution.

![1](https://github.com/cardonabits/haxo-hw/assets/164619675/90faca12-4fff-44fa-93aa-ffc8f7a3711e)

[https://www.thingiverse.com/thing:6562738](url)

There it is.

**Comment by jcard0na** [comment]
Thank you for sharing.  Looks cool!
